rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,136 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# TL Quarterly Planning Process
+
+## Summary
+
+This document details the a quarterly process by which TLs (Technical Leads) will contribute to the overall DCP
+quarterly roadmapping process via a Technical Objectives RFC, presented during the TL+PM Planning Call.
+
+## Author and Shepherd
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+## Motivation
+
+The new Roadmaps and Planning Process, detailed in the RFC [here](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/text/0000-roadmaps%2Bplanning.md),
+defines a process for annual roadmaps that inform quarterly release planning. TLs (Technical Leads) should contribute
+“grassroots” knowledge about the technical requirements of the DCP on a quarterly basis to guide the 
+[quarterly Release process](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/text/0000-roadmaps%2Bplanning.md#planning-and-executing-a-release)
+via a Technical Objectives RFC. This document details the contents of the Technical Objectives RFC as well as the
+process that must occur such that this artifact is drafted and curated according to a timeline that will allow us to
+present the artifact to the quarterly TL+PM objective curation meeting.
+
+This proposed TL Quarterly Planning Process is **owned by the collection of TLs in the DCP** and is intended to
+be a live process that iterates over time as required. The first iteration of the planning process is detailed below.  
+
+Note: The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT
+RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, RFC2119, and RFC8174
+when, and only when, they appear in all capitals, as shown here.
+
+### User Stories
+
+As a TL of the DCP, I am able to surface important technical priorities to Product Management that will be incorporated
+into the yearly roadmap. 
+
+As a developer on the DCP, I am able to allocate enough time to work on high priority technical architectural issues
+knowing that the work will facilitate quicker development of feature requests in the long term as informed by
+the roadmap themes.
+
+## Definitions
+**Quarter:** One quarter is equal to 13 weeks. Generally the quarters have the following timeline:
+
+- Quarter 1 (Q1): first Monday of January -> last Friday of March
+- Quarter 2 (Q2): first Monday of April -> last Friday of June
+- Quarter 3 (Q3): first Monday of July -> last Friday of September
```

Comment:
It doesn't really matter, but this is kind of a weird way to define quarters. For example, this year it would leave a 9 day intra-quarter period between Q3 and Q4.

Reply:
Removed it altogether since the Roadmap and Planning RFC doesn't contain it.